services/api/test_main.py

```diff
@@ -0,0 +1,70 @@
+from fastapi.testclient import TestClient
+from main import app, db
+import re
+import json
+
+
+client = TestClient(app)
+names = [
+    "Adam",
+    "aLICE",
+    "airplane",
+    "CAR",
+    "adaM",
+    "toy",
+    "fLOwer",
+    "adam",
+    "family",
+]
+
+
+def test_get_items_return_json():
+    response = client.get("/")
+    assert response.status_code == 200
+    assert type(response.json()) == str
+    assert re.search(r"\[.*]", response.json()) is not None
+
+
+def test_search():
+    db.drop()
+    for element in names:
+        db.add({"name": element})
+
+    assert db.search("a") == [
+        "Adam",
+        "aLICE",
+        "airplane",
+        "CAR",
+        "adaM",
+        "adam",
+        "family",
+    ]
+    assert db.search("fl") == ["fLOwer"]
+    assert db.search("z") == []
+    assert db.search("") == names
```

Comment:
każdy z tych assertów to powinnien być inny test i podając odpowiednią nazwę metodzie jeżeli któryś z nich by się wysypał to łatwiej sprawdzić co jest nie tak